requirements-dev.txt

```diff
@@ -9,3 +9,4 @@ moto
 pytest
 requests>=2.22.0
 sqlalchemy
+chalice
```

Comment:
You'll wanna remove `chalice` from `dcp_prototype/backend/browser/requirements-dev.txt` then

Reply:
you're right. I was using it previously, but decided to go with a simpler testing approach.